_upload_data.py

```diff
@@ -0,0 +1,72 @@
+import click
+import os
+from osfclient.api import OSF
+
+# this script does the same as (from terminal)
+# osf -r -p your_password -u your_username upload local_path remote_path
+
+LOCAL_PATH = 'meg/data'  # local path to the data
```

Comment:
Why not just `data`?

Reply:
this is a variable which needs to be set by a user depending where he/she stores the data. `can be `data` or anything else